tests/testthat/test-multitables.R

```diff
@@ -245,8 +246,9 @@ with_mock_crunch({
             test_that("tab book print methods", {
                 ## TODO: print method for TabBookResult
             })
+
             test_that("The first result in a MultitableResult has 3 dimensions", {
-                expect_identical(dim(book[[1]][[1]]), c(5L, 1L, 2L))
+                expect_equal(dim(showMissing(book[[1]][[1]])), c(5, 1, 2))
```

Comment:
Same question as above: any reason to not use `expect_identical` and `c(5L, 1L, 2L)`?